src/containers/ftp/ftp_files.rb

```diff
@@ -101,6 +103,7 @@ def upload_file(uri, local_file, options = {})
         ftp.binary = options[:binary].nil? ? true : options[:binary]
         ftp.open_timeout = nil # default is nil
         ftp.read_timeout = 60 # default is 60
+        ftp.ssl = options[:use_tls]
```

Comment:
it's slightly confusing to name the option `use_tls`, when the ruby option is `ssl`, but it seems fine.

I'm not sure if the "check the schema" logic belongs here or in the `ftp.rb` class - the full uri with schema is available here as well.